pom.xml

```diff
@@ -28,7 +28,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>2.5</version>
+        <version>2.14</version>
```

Comment:
To use `assertEqualDataBoundBeans` supporting `DataBoundSetter` setters in jenkins-test-harness 2.11.
(Not support for `DataBoundSetter` members yet...)